app/package.json

```diff
@@ -47,7 +45,7 @@
 		"bestzip": "^2.1.2",
 		"coveralls": "3.1.0",
 		"cross-env": "7.0.2",
-		"node-sass": "^7.0.1",
+		"node-sass": "^8.0.0",
```

Comment:
node-sass is deprecated, use sass instead ?

Reply:
@ddecrulle good catch. I have just migrate to sass. Thanks .